media-gfx/librecad/files/librecad-2.1.3-CVE-2021-45342.patch

```diff
@@ -0,0 +1,35 @@
+From 4edcbe72679f95cb60979c77a348c1522a20b0f4 Mon Sep 17 00:00:00 2001
```

Comment:
Please add a link to where you got it from and also any relevant Gentoo bugs.

Reply:
fixed